src/pages/general/login/actions/spec/loginRequest.spec.ts

```diff
@@ -3,7 +3,7 @@ import ReduxThunk from 'redux-thunk';
 import configureStore from 'redux-mock-store';
 import { FormValidationResult, FieldValidationResult } from 'lc-form-validation';
 import { navigationHelper } from '../../../../../common/helper/navigationHelper';
-import { loginApi } from '../../../../../rest-api/login/loginAPI';
+import * as loginApi from '../../../../../rest-api/login';
```

Comment:
this line change is inconsistent with the change in `loginRequest.ts` where you rename `loginApi` to `login`, so, I guess this should be renamed to `login` too.